src/web/components/LeftColumn.stories.tsx

```diff
@@ -1,17 +1,25 @@
 import React from 'react';
+import { css } from 'emotion';
 
 import { Flex } from '@root/src/web/components/Flex';
 import { RightColumn } from '@root/src/web/components/RightColumn';
 import { ArticleContainer } from '@root/src/web/components/ArticleContainer';
 import { Section } from '@frontend/web/components/Section';
 
+import { palette } from '@guardian/src-foundations';
```

Comment:
We migrated away from this as we now have our own `palette`

Reply:
Re this import this was deprecated by Simon et al a while back, I think for performance reasons, not 100% sure, but I remember doing a big mob to remove them all